src/basket/base.py

```diff
@@ -88,42 +87,31 @@ def parse_response(res):
     try:
         result = json.loads(res.content)
     except Exception:
-        log.exception("Error parsing JSON returned by basket (%s)" % res.content)
+        log.exception(f"Error parsing JSON returned by basket ({res.content})")
```

Comment:
This is another instance of string interpolation in logging. I would set this to:

```suggestion
        log.exception("Error parsing JSON returned by basket (%s)", res.content)
```

Reply:
If I understand, though, this isn't an exception being logged, but the string that caused the exception. So in this instance this makes sense?